### ISSUE TITLE
Hi, I read the "why no MNK?"

### ISSUE THREAD

**Issue by Ghost-of-Razgriz** [attickdoor/XIVComboPlugin#246]
I agree with the sentiment, hell, felt the same way before reading it. Though what about the aoe combo? It's pretty much the same as anyone else's. Yeah it can combo in to a ST combo but because there's no pcombo buttons for it, it doesn't matter. It doesn't have the decision making that the ST combos do. At best, you can treat PB and Form Shift as you do Swiftcast on RDM. If you're worried about Four-Point Fury, there is a similar concern with SAM anyway and I just moved Jinpu and Shifu to my 3rd bar. I generally don't have to touch them, but they're there just in case. This is in reference to Meikyo Shisui.

To me, this was always about combating button bloat. Which is a problem I very much suffer with MNK. I don't use this mod to subvert being a brainlet. But I think you could probably have it open AotD>Fpf>Rockbreaker with Form Shift/PB changing it or being ignored. Form Shift uses Fpf and PB uses Rockbreaker, why? Form Shift would only be used in an aoe situation to get the damage % rolling, PB would be used to spam the highest potency move. Only in an AoE situation, which is all this recommendation is for.

If you still find this compromise to be beyond what you desire for Pcombo I understand.

**Comment by KaguyaEne** [comment]
This about it this way: Every class in the game has a linear combo line, ST 1-2-3 or AoE 1-2(-3), but Monk does not. It's the only class in the game to have entirely non-linear combos that can be done interchangeably between AoE and ST. If it was just that then sure it can be a feasible option because it's linear and is just a 1-2-3. Perfect Balance and Formless Fist by themselves breaks linear logic and is no longer a simple 1-2-3 but now branches into 14 variants for ST and 3 variants for AoE just for Formless Fist. Perfect Balance has far more variants with 3 different Masterful Blitz attacks. And trying to counter argue with "well Perfect Balance can just spam the highest potency GCD" means nothing when it doesn't return any buttons doing so.

TLDR; When faced with choice, that isn't linear, the plugin can't, and won't, help because otherwise it's much closer to a bot.

**Comment by Ghost-of-Razgriz** [comment]
I don't see how that refutes my "Only for AoE" consideration. I am not and would not ask for a ST pcombo rotation. Tbf I don't understand your last sentence before the TLDR. AoE is a pretty linear decision unlike the ST combo.

Cheers ^^

**Comment by lovelydumpling** [comment]
I'm not sure what happened with the AoE combo either. It used to be there but not anymore. Shame cause of button bloat but I'll have to manage. One of these days XIVCombo might not be around anymore so I suppose I can get used to at least one job without it.
